main.go

```diff
@@ -1,16 +1,21 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/serenize/snaker"
+	"test-ci/lib"
+)
 
 func main() {
 	var s string
 
-	if s, _ := hoge(); s != "" {
+	if s, _ := hello(); s != "" {
 		return
 	}
 	fmt.Println(s)
 }
 
-func hoge() (string, error) {
-	return "hello", nil
+func hello() (string, error) {
+	return snaker.CamelToSnake(lib.GetStr()), nil
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: snaker
* [gotype] undeclared name: lib
* [aligncheck] undeclared name: lib
* [unconvert] undeclared name: lib
* [structcheck] unused struct field undeclared name: lib
* [errcheck] error return value not checked (undeclared name: lib)
* [varcheck] unused global variable undeclared name: lib

<!-- Comment by [SideCI](https://sideci.com) -->